lib/forking_test_runner.rb

```diff
@@ -146,8 +157,14 @@ def run_test(file)
         preload_fixtures
         ActiveRecord::Base.connection.disconnect!
       end
+      output = nil
+
       change_program_name_to file do
+        rpipe, wpipe = IO.pipe
+
         child = fork do
+          rpipe.close
+          $stdout.reopen(wpipe)
```

Comment:
this will suppress all output until the test is complete, can we only do this if we are not in verbose mode ?